src/components/BottomSheet.tsx

```diff
@@ -0,0 +1,261 @@
+import React, { useMemo, useRef, useCallback, useState } from 'react';
+import { Text, TouchableOpacity, StyleSheet, View } from 'react-native';
+import colors from '@styles/colors';
+import BottomSheet, { BottomSheetScrollView } from '@gorhom/bottom-sheet';
+import { GestureHandlerRootView } from 'react-native-gesture-handler';
+import { WeatherComponent } from '@components/WeatherComponent';
+import { ExtraInfoType } from '@components/ExtraInfo';
+import { WarningType } from '@components/Warning';
+import { WeatherType } from '@components/WeatherIcon';
+import { MAIN_HORIZONTAL_OFFSET } from '@styles/constants';
+import { BottomSheetButtons } from '@utils/constants';
+
+export const Sheet = () => {
+    const sheetRef = useRef<BottomSheet>(null);
+
+    const snapPoints = useMemo(() => ['29%', '96%'], []);
+
+    const handleSheetChanges = useCallback((index: number) => {
+        console.log('handleSheetChanges', index);
+    }, []);
+
+    const [sectionWeather, setSectionWeather] = useState(true);
+    const [sectionСlothes, setSectionСlothes] = useState(false);
+
+    const switchWeather = () => {
+        if (sectionWeather) {
+            setSectionWeather(false);
+        } else {
+            setSectionWeather(true);
+            setSectionСlothes(false);
+        }
+        if (sectionWeather === true) {
+            setSectionWeather(true);
+        }
+    };
+
+    const switchСlothes = () => {
+        if (sectionСlothes) {
+            setSectionСlothes(false);
+        } else {
+            setSectionСlothes(true);
+            setSectionWeather(false);
+        }
+        if (sectionСlothes === true) {
+            setSectionСlothes(true);
+        }
+    };
+
+    if (sectionWeather) {
```

Comment:
Do not use if like that: you have the same elements in return, so it is better to do like:
``` tsx
<BottomSheet>
{sectionIsWeather ? (<></>) : (<></>)}
</BottomSheet
```